backend/app/infrastructure/database/user_antisybil.py

```diff
@@ -0,0 +1,31 @@
+from typing import Optional
+from datetime import datetime
+import json
+
+from app.infrastructure.database.models import GPStamps
+from app.infrastructure.database.user import get_by_address, add_user
+from app.exceptions import UserNotFound
+from app.extensions import db
+
+
+def add_score(user_address: str, score: str, expires_at: datetime, stamps) -> GPStamps:
```

Comment:
`stamps` type is missing

Reply:
It is very complex and never actually used. No point documenting it.

Reply:
isn't it just `list[dict]`? or `dict`? It doesn't have to be very precise

Reply:
:+1: